### PR TITLE
[fix][ci][branch-2.11] Fix broken CI

### PR DESCRIPTION
**Pull request by nodece**
### Motivation

Error 1:
```
Error:  Failures: 
  Error:  org.apache.pulsar.broker.service.BrokerServiceTest.testGetTopic(org.apache.pulsar.broker.service.BrokerServiceTest)
  [INFO]   Run 1: PASS
  Error:    Run 2: BrokerServiceTest.testGetTopic:1492 expected [0] but found [1]
  [INFO] 
```

Error 2:
```
leveldb-0.10.jar unaccounted for in lib/presto/LICENSE
leveldb-api-0.10.jar unaccounted for in lib/presto/LICENSE
leveldb-0.[12](https://github.com/apache/pulsar/actions/runs/4239032335/jobs/7399965467#step:11:13).jar mentioned in lib/presto/LICENSE, but not bundled
leveldb-api-0.12.jar mentioned in lib/presto/LICENSE, but not bundled
```

### Modifications

- Fix test
- Fix presto license

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by nodece** [comment]
ping @Technoboy-, the CI has been passed.

**Review by Technoboy- — APPROVED** [review]

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java [hunk 1]
```diff
@@ -956,13 +956,18 @@ public CompletableFuture<Optional<Topic>> getTopic(final String topic, boolean c
 
     public CompletableFuture<Optional<Topic>> getTopic(final String topic, boolean createIfMissing,
                                                        Map<String, String> properties) {
+        return getTopic(TopicName.get(topic), createIfMissing, properties);
+    }
+
+    public CompletableFuture<Optional<Topic>> getTopic(final TopicName topicName, boolean createIfMissing,
```
Comment (tisonkun) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java:962]: @nodece may you talk a bit detail how this change fixes the issue? And if a similar issue is in the master branch.

Comment (nodece) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java:962]: See https://github.com/apache/pulsar/pull/17416

**Pull request merged**